### ISSUE TITLE
Smooth out learning spikes

### ISSUE THREAD

**Issue by gordonwatts** [gordonwatts/CalRatioTrainer#20]
Current loss plot has a lot of spikes - much worse than we are used to seeign with Alex's training. This needs to be solved.

**Comment by gordonwatts** [comment]
The adjustable LR was causing real problems - this is lots better. Other things to try:

* Adjust the default learning rates - what if we made them a very small amount larger?
* Implement gradient clipping so when a vanishing gradient does happen, we can limit the damage.

Also, might be worth carefully looking at adversary loss vs main network loss to understand if this is coming from a particular place.
